Packs/CommonScripts/Scripts/Base64EncodeV2/Base64EncodeV2.py

```diff
@@ -0,0 +1,22 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import base64
+
+
+def encode(input):
+    input_bytes = input.encode('utf-8')
+    res = base64.b64encode(input_bytes)
+    outputs = {
+        'Base64': {
+            'encoded': res
+        }
+    }
+
+    return res, outputs
+
+
+if __name__ in ('builtins', '__builtin__'):
+    input = demisto.args().get('input')
+    return_outputs(*encode(input))
```

Comment:
shouldn't you add try/catch for later errors not showing the stacktrace ina warrroom entry?